src/Database/index.d.ts

```diff
@@ -16,7 +16,7 @@ import { $ReadOnlyArray, $Exact } from '../types'
 
 type DatabaseProps = $Exact<{
   adapter: DatabaseAdapter
-  modelClasses: Model[]
+  modelClasses: typeof Model[]
```

Comment:
this is causing problems when doing `new Database({adapter, models})` where models is of type `Model[]`

Reply:
that's odd... What error do you get ? 

I was getting an error _before_ making this change...

btw it should be `new Database({adapter, modelClasses})`

Reply:
I don't really know TypeScript, but it surely can't be `Model[]` -- should't it be `class<Model>[]`?

Reply:
I've reverted to the original custom `Class` type just to be safe. I suspect there may be something with different typescript versions...